libs/utils/src/ballot_styles.ts

```diff
@@ -141,11 +130,36 @@ export function getDefaultLanguageBallotStyles(
  *
  *
  */
-export function getParentBallotStyles(
+
+export function getGroupedBallotStyles(
```

Comment:
Nit: I think `getBallotStyleGroups` would be a slightly clearer name.

Also need to update comment for this function.